components/admin_console/admin_definition.jsx

```diff
@@ -1411,6 +1411,14 @@ export default {
                         help_text: t('admin.support.emailHelp'),
                         help_text_default: 'Email address displayed on email notifications and during tutorial for end users to ask support questions.',
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'SupportSettings.TermsOfServiceLink',
+                        label: t('admin.support.termsTitle'),
+                        label_default: 'Terms of Service link:',
```

Comment:
Sorry, I meant using `Link` instead of `link`. E.g. `Terms of Service Link`. 

Similarly for the other settings.

Reply:
ahhh it was the other way around.. well then I need to change this in various places then, will do and update the PR